app/i18n/ar-SA/common.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Scene": "Scene",
+  "Scenes": "Scenes",
```

Comment:
Does our localization system have a proper pluralization system?  I think some languages have multiple plural forms depending on whether it is "many" or "few".

Reply:
This plugin has some kind of pluralization http://kazupon.github.io/vue-i18n/en/pluralization.html. However, I prefer to avoid this stuff to not to make translations overcomplicated. In addition for example in the Russian language pluralization can be done only with help of very big dictionaries because the ending of a plural word depends not only how much things you have, it also depends on what kind of things you have (each word have gender `he`, `she` and 'it' that play a big role in pluralization). And some other languages have the same situation. So this is a big pain in ass and  instead `You have 5 cars` for multilingual projects I suggest to use `Your cars amount: 5`

Reply:
Cool sounds good.  You're the expert.  I never really know what looks/sounds weird in other languages and what doesn't.